lib/recurly/base.php

```diff
@@ -173,6 +173,7 @@ public function getLinks() {
     'adjustment' => 'Recurly_Adjustment',
     'adjustments' => 'Recurly_AdjustmentList',
     'coupon' => 'Recurly_Coupon',
+    'unique_coupon_codes' => 'Recurly_UniqueCouponList',
```

Comment:
yeah i feel like for everything else there's a pretty clear 1:1 mapping so i'd like to keep it as consistent as possible... in spite quirks like of `Recurly_Addon` and `Recurly_Tax_Detail`.